code/datums/jobs/squads.dm

```diff
@@ -83,6 +84,11 @@ GLOBAL_LIST_EMPTY(helmetmarkings_sl)
 	var/image/armors1 = image('icons/mob/suit_1.dmi',icon_state = "std-armor")
 	armor.color = color
 	armors1.color = color
+	var/image/gloves = image('icons/mob/hands.dmi',icon_state = "std-gloves")
+	var/image/gloves1 = image('icons/mob/hands.dmi',icon_state = "std-gloves")
+	gloves.color = color
+	gloves1.color = color
```

Comment:
is `gloves1` needed here?

Reply:
I'll check.

Reply:
Removed the var, encountered no issues or runtimes from local testing.

Reply:
Remove it again? This variable shouldn't be needed.